Assets/HoloToolkit/Input/Scripts/Utilities/MotionControllerVisualizer.cs

```diff
@@ -234,82 +239,124 @@ private void InteractionManager_InteractionSourceLost(InteractionSourceLostEvent
+            while (modelTask.Status == AsyncStatus.Started)
+            {
+                yield return null;
+            }
 
-                if (modelStream == null)
-                {
-                    Debug.Log("Model stream is null.");
-                    yield break;
-                }
+            IRandomAccessStreamWithContentType modelStream = modelTask.GetResults();
 
-                if (modelStream.Size == 0)
-                {
-                    Debug.Log("Model stream is empty.");
-                    yield break;
-                }
+            if (modelStream == null)
+            {
+                Debug.Log("Model stream is null, loading alternate.");
+                LoadAlternateControllerModel(source);
+                yield break;
+            }
 
-                byte[] fileBytes = new byte[modelStream.Size];
+            if (modelStream.Size == 0)
+            {
+                Debug.Log("Model stream is empty, loading alternate.");
+                LoadAlternateControllerModel(source);
+                yield break;
+            }
 
-                using (DataReader reader = new DataReader(modelStream))
-                {
-                    DataReaderLoadOperation loadModelOp = reader.LoadAsync((uint)modelStream.Size);
+            byte[] fileBytes = new byte[modelStream.Size];
 
-                    while (loadModelOp.Status == AsyncStatus.Started)
-                    {
-                        yield return null;
-                    }
+            using (DataReader reader = new DataReader(modelStream))
+            {
+                DataReaderLoadOperation loadModelOp = reader.LoadAsync((uint)modelStream.Size);
 
-                    reader.ReadBytes(fileBytes);
+                while (loadModelOp.Status == AsyncStatus.Started)
+                {
+                    yield return null;
                 }
 
-                controllerModelGameObject = new GameObject();
-                GLTFComponentStreamingAssets gltfScript = controllerModelGameObject.AddComponent<GLTFComponentStreamingAssets>();
-                gltfScript.ColorMaterial = GLTFMaterial;
-                gltfScript.NoColorMaterial = GLTFMaterial;
-                gltfScript.GLTFData = fileBytes;
+                reader.ReadBytes(fileBytes);
+            }
+
+            controllerModelGameObject = new GameObject();
+            GLTFComponentStreamingAssets gltfScript = controllerModelGameObject.AddComponent<GLTFComponentStreamingAssets>();
+            gltfScript.ColorMaterial = GLTFMaterial;
+            gltfScript.NoColorMaterial = GLTFMaterial;
+            gltfScript.GLTFData = fileBytes;
 
-                yield return gltfScript.LoadModel();
+            yield return gltfScript.LoadModel();
+            FinishControllerSetup(controllerModelGameObject, source.handedness.ToString(), source.id);
 #else
-                yield break;
+            yield break;
```

Comment:
This should also have `LoadAlternateControllerModel(source);` right before the `yield break;`, that way the alternate models will load in the editor if needed. Right now, nothing will load in the editor.